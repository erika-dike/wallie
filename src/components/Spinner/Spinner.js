import React from 'react';

import './Spinner.css';

/*
Tiny component for rendering a spinner animation
*/
const Spinner = () =>
  <div>
    <div className="overlay" />
    <span className="spotlight spinner">
      <i className="fa fa-spinner fa-pulse fa-2x fa-fw" />
      <span className="sr-only">Loading...</span>
    </span>
  </div>;

export default Spinner;
