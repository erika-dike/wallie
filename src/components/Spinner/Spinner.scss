.spinner {
    margin-top: 8px;
    margin-left: -11px;
    color: #fff;
}
