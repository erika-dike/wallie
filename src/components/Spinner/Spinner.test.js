import React from 'react';
import renderer from 'react-test-renderer';

import Spinner from './Spinner';


it('renders correctly', () => {
  const tree = renderer.create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
