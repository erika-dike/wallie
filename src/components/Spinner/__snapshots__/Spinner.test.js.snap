exports[`test renders correctly 1`] = `
<div>
  <div
    className="overlay" />
  <span
    className="spotlight spinner">
    <i
      className="fa fa-spinner fa-pulse fa-2x fa-fw" />
    <span
      className="sr-only">
      Loading...
    </span>
  </span>
</div>
`;
